### ISSUE TITLE
Subtrees of multiple nodes

### ISSUE THREAD

**Issue by mgornick** [stefankroes/ancestry#75]
I was wondering if there are any plans to support a class method to return the subtrees of multiple nodes.

In this example of 3 trees:

A => B
C => D
E => F

```
>> Node.subtrees_for([A, B])
=> [A, B, C, D]
```

**Comment by kbrock** [comment]
Believe it or not, still working on this one.
Hoping I'm close.

**Comment by nflorentin** [comment]
Rails 5 added support for "or" queries, so it could be something like that using only 2 queries:
```
nodes = Node.all # first query here
query = nodes[0].subtree
nodes[1..-1].each do |n|
  query = query.or(n.subtree)
end
query.load # second query here
```

But I think there is a more elegant way to write it...

**Comment by kbrock** [comment]
@nflorentin yes, using ActiveRecord `or` would get rid of a bunch of arel helpers.

I'm thinking the next major version of ancestry will probably drop older versions of active record that do not support `or`.
